redfish-core/lib/FabricAdapters.hpp

```diff
@@ -359,5 +360,58 @@ inline void requestRoutesFabricAdapters(App& app)
                 BMCWEB_LOG_DEBUG << "Adapter =" << fabricAdapter;
                 getAdapter(asyncResp, fabricAdapter);
             });
+
+    BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/FabricAdapters/<str>/")
+        .privileges({{"Login"}})
+        .methods(boost::beast::http::verb::patch)(
+            [](const crow::Request& req,
+               const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+               const std::string& fabricAdapter) {
+                std::optional<bool> locationIndicatorActive;
+                if (!json_util::readJson(req, asyncResp->res,
+                                         "LocationIndicatorActive",
```

Comment:
I don't see LocationIndicatorActive here https://redfish.dmtf.org/schemas/FabricAdapter.v1_3_0.json so this doesn't work and will fail the validator. 
I created a PR in Redfish to add this but won't be  out until Aug. If we need this before, this will need to be OEM 

                "Links": {
                    "$ref": "#/definitions/Links",
                    "description": "The links to other Resources that are related to this Resource.",
                    "longDescription": "The Redfish Specification-described Links Property shall contain links to Resources related to but not subordinate to this Resource."
                },
                "Location": {
                    "$ref": "http://redfish.dmtf.org/schemas/v1/Resource.json#/definitions/Location",
                    "description": "The location of the fabric adapter.",
                    "longDescription": "This property shall contain location information for the fabric adapter.",
                    "versionAdded": "v1_1_0"
                },
                "Manufacturer": {
                    "description": "The manufacturer or OEM of this fabric adapter.",
                    "longDescription": "This property shall contain a value that represents the manufacturer of the fabric adapter.",
                    "readonly": true,
                    "type": [
                        "string",
                        "null"
                    ]
                },